pkgs/applications/networking/browsers/falkon/default.nix

```diff
@@ -21,8 +24,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     libpthreadstubs libxcb libXdmcp
+    qtsvg qttools qtwebengine qtx11extras qtwayland
```

Comment:
this should just be added on stdenv.isLinux

Reply:
Thanks. I changed this, but it doesn't seem to be activating on my system, despite me running NixOS on an x64 processor? Or I didn't add it correctly, but I followed the pattern I saw in other files.